db/migrate/20220120092504_create_users.rb

```diff
@@ -5,8 +5,6 @@ class CreateUsers < ActiveRecord::Migration[6.1]
   def change
     create_table :users do |t|
       t.integer :status, null: false, default: 0
-      t.timestamp :net_state, null: false
```

Comment:
we don't change migrations if they are already in develop. Your team have their local DB setup, and they won't get your updates because you change migration with a timestamp  (20220120092504) which is already run by your team mates. So Rails won't know that you have changes to schema unless you create new migration. Create new migration where you remove these fields from users table